tests/shouldwork/Signal/BiSignal/T2472.hs

```diff
@@ -0,0 +1,44 @@
+module T2472 where
+
+import Clash.Prelude
+import Clash.Annotations.SynthesisAttributes
+
+import Data.List (isInfixOf)
+import qualified Prelude as P
+import System.Environment (getArgs)
+import System.FilePath ((</>))
+
+-- BiSignalIn must be present at the top level.
+-- Putting it inside a multifield record caused the assign to drive an internal wire.
+type FtdiIO dom = "FTDI_D" ::: BiSignalIn 'Floating dom 8
+                  `Annotate` 'StringAttr "LOC" "P1, P2, P4, N5, P5, M6, N6, M7"
+                  `Annotate` 'StringAttr "IOSTANDARD" "LVTTL"
+                  -- FIXME: Annotations break BiSignal in Clash 1.6.4!
```

Comment:
Could you remove this line? It's confusing if we ever look at this test later :-)